bambi/plots/effects.py

```diff
@@ -1,18 +1,21 @@
-# pylint: disable = protected-access
-# pylint: disable = too-many-function-args
-# pylint: disable = too-many-nested-blocks
 from dataclasses import dataclass, field
 import itertools
 from typing import Dict, Union
 
 import arviz as az
 import numpy as np
 import pandas as pd
+from pandas.api.types import is_categorical_dtype, is_string_dtype
```

Comment:
Could you sort the imports so this `from ...` appears below all the `import x as y`?